internal/template/templates/.claude/agents/moai/plan-auditor.md

```
@@ -0,0 +1,269 @@
+
+### Group 6: Consistency
+
+- CN-1: No two requirements contradict each other
+- CN-2: Exclusions do not conflict with included requirements
+- CN-3: Priority and labels are consistent with the stated scope
+
+## Output Format
+
+Write the audit report to `.moai/reports/plan-audit/{SPEC-ID}-review-{iteration}.md`.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)